src/data/queries/tests/__snapshots__/newsStory.test.tsx.snap

```diff
@@ -362,7 +362,7 @@ exports[`node--news_story formatData outputs formatted data 1`] = `
     "width": 456,
   },
   "introText": "When a hospital has a host of great doctors, honoring just two every year is challenging. ",
-  "lastUpdated": "2019-05-14T15:35:12+00:00",
+  "lastUpdated": "2020-03-24T20:10:28+00:00",
```

Comment:
Because `lastUpdated` was changed in `src/lib/drupal/query.ts`, this value changed as well.